### PR TITLE
Refactor: making cli descriptions more complete

### PR DESCRIPTION
**Pull request by heliomar-pena**
Now it's more informative, not sure if it is more clear or if is comfortable for reading.

What do you think? @Israel-Laguan @YoelFerreyra 

NOW:
image

BEFORE:
image

### CODE REVIEW

**Review by Israel-Laguan — APPROVED** [review]

File: cli.js [hunk 1]
```diff
@@ -19,16 +20,30 @@ program
...
+  )
+  .option(
+    "-n, --name <string>",
+    "The name of the property that you want your text has on the output files.",
+  )
   .option(
     "-e, --engine <string>",
-    "[OPTIONAL]. Engine to use for the translation. In case you don't define it will be use by default all the translation engines that are free and doesn't requires API Key.",
+    `[OPTIONAL]. The engine that you want to use to translate the text. Available options: ${validEngines.join(", ")}
+    If you specify a engine on the command, it will put it on the first position of the array of engines.
```
Comment (Israel-Laguan) [inline cli.js:38]: In the example have some extra space at the beginning of the line, maybe check alternatives for better readability

Comment (heliomar-pena) [reply cli.js:38]: will take a look. Thanks!

Comment (heliomar-pena) [reply cli.js:38]: not sure if this is better?

idea 1:
image

idea 2:
image

or if you have any other idea you wanted try?

Comment (heliomar-pena) [reply cli.js]: > not sure if this is better?
> 
> idea 1: image
> 
> idea 2: image
> 
> or if you have any other idea you wanted try?

@YoelFerreyra can you help us to decide which looks better and is easier to read?

Comment (Israel-Laguan) [reply cli.js]: I like number 1

**Pull request merged**